tools/pleasedontkillme/main.go

```diff
@@ -0,0 +1,25 @@
+package main
```

Comment:
Thanks for adding this interesting test, I think we should come up with a better name, maybe "not_ending_proc" and and lets choose a better  path for it than the root folder of minikube
we generally wanna keep the root folder clean and make it more oragnized.

, how about consider putting it under "test" folder or maybe "hack folder"

Reply:
There you go :)

I wasn't sure because there was no such thing in the repo..
so I tried [this](https://github.com/golang-standards/project-layout#tools) and hoped for the best